modules/apps/frontend-icons/frontend-icons-web/src/main/resources/META-INF/resources/js/pages/instance_settings/AddIconPackModal.tsx

```diff
@@ -50,6 +66,15 @@ export default function AddIconPackModal({
 
 		const formData = new FormData();
 
+		if (
+			!(
+				svgFileInputRef?.current?.files &&
+				svgFileInputRef?.current?.files[0]
```

Comment:
When using, 

```js
const {current: {files = []}} = svgFileInputRef; 
```

and then using just 

```
if(!rules) {
...
```

it will have the same effect and will fail silently if current is not defined or maybe svgFileInputRef? 🤔

Reply:
So I think the only problem here is that `current` can be null, so we can't be sure that it will always be object like unless we provide an initial value

Reply:
Ohh, yes! Makes sense!